modules/shortcodes/instagram.php

```diff
@@ -132,6 +132,79 @@ function jetpack_instagram_embed_reversal( $content ) {
 	return $content;
 }
 
+/**
+ * List of whitelisted and sanitized parameters
```

Comment:
Should we change this to `safelisted` or simply `allowed`? (see e.g. #16100 or #16109)

```suggestion
 * List of allowed and sanitized parameters
```